build.rs

```diff
@@ -86,19 +110,21 @@ fn build_data() -> InputData {
                 None => unreachable!("no colon found in line {}", i),
             };
             let code_point = line[..colon_pos].trim();
-            let pinyin = line[colon_pos + 1..].trim();
-            // 检查我们理解拼音数据中的每个字符
-            for ch in pinyin.chars() {
-                let is_known = ch.is_ascii()
-                    || PHONETIC_SYMBOL_MAP.iter().any(|(c, _, _)| *c == ch)
-                    || TONE_MAP.iter().any(|(c, _)| *c == ch)
-                    || NON_PINYIN_TONE.iter().any(|c| *c == ch);
-                assert!(
-                    is_known,
-                    "unknown character {:?} at line {}: {}",
-                    ch, i, line,
-                );
+            let pinyin_list: Vec<&'static str> = line[colon_pos + 1..].trim().split(',').collect();
+
+            // 确保输入数据的字符全部在我们预料之中。
+            // 同时也可以提前知道一些被遗弃的码位，如: U+E7C8 和 U+E7C7
+            for pinyin in &pinyin_list {
```

Comment:
我个人比较推荐下面这种写法
```suggestion
            for pinyin in pinyin_list.iter() {
```

用 `&pinyin_list` 的话依赖 `IntoIterator for &[T]`……可能也没什么大问题……

Reply:
已更改。